### PR TITLE
Demand TLS creatives

### LINKED ISSUE #35: Demand HTTPS creatives
Most sites want to migrate to HTTPS, if they haven't already. The biggest problem for almost all the sites, are that the creatives are not HTTPS compliant. 

In my opinion, we should start to demand https creatives in this spec already now, so the rollout for more secure sites will be smoother. 

For sites on http, this won't be a problem. Its the other way thats a problem, https-sites with http creatives.

### PR DESCRIPTION
**Pull request by OleSan**
closes #35

### CODE REVIEW

File: specification.md [hunk 1]
```diff
@@ -19,18 +19,18 @@ This is the specification for both the HTML banners and image banners. Inline is
 Note: The size limitation is on the initial load. You can lazy load additional content, but for the first rendering this is the limit.
 
 ## 2. Limitations
-
+* The HTML-file delivered should just be _one file_ with all CSS required for the ad inline in the HTML.
+* References to resources must be absolute and start with https:/ with TLS (not SSL).
```
Comment (gregersrygg) [inline specification.md]: should be `https://` (missing a /)

File: specification.md [hunk 2]
```diff
@@ -19,18 +19,18 @@ This is the specification for both the HTML banners and image banners. Inline is
...
+ * Check for a list of [best praticses](https://www.ssllabs.com/ssltest/index.html ) and an easy [server test](https://www.ssllabs.com/projects/best-practices/index.html).
```
Comment (gregersrygg) [inline specification.md]: How about:
```
The servers used must receive [A-](https://www.ssllabs.com/downloads/SSL_Server_Rating_Guide.pdf) or better in SSL Lab´s [Server Test](https://www.ssllabs.com/projects/best-practices/index.html). See the Server Test result and [SSL Lab´s Best Practices](https://www.ssllabs.com/ssltest/index.html) for tips on what to improve.
```

File: specification.md [hunk 3]
```diff
@@ -19,18 +19,18 @@ This is the specification for both the HTML banners and image banners. Inline is
...
 	~~`<meta name="viewport" content="width=devicewidth">`~~
 
-* The [Geo Location APIs](spec/geoapi.md) or other "HTML5"-APIs that require user confirmation, can only be used after a user interaction.
-* The HTML-file delivered should just be _one file_ with all CSS required for the ad inline in the HTML.
 * [Maximum of two HTTP requests to JavaScript libraries (at least one to CDN).](spec/maximumhttprequests.md)
 * Animation _prior to a user interaction_ must be written using [CSS3 Transitions, Transforms and/or Animation](spec/cssforanimations.md) or with GSAP library hosted on Cloudflare CDN.
  * [Non GSAP JavaScript animations are forbidden before an user interaction](spec/jsanimations.md).
 * Javascript console methods are not allowed
-* References to resources must be absolute and start with http:// or https:// , not only //. Because of limitations in our delivery system.
+* The [Geo Location APIs](spec/geoapi.md) or other "HTML5"-APIs that require user confirmation, can only be used after a user interaction.
```
Comment (gregersrygg) [inline specification.md]: I don't think this was supposed to be in this pull-request ;)

Comment (OleSan) [inline specification.md]: I was only moving it down a bit in the list, to group similar items together.

**Comment by gregersrygg** [comment]
Looking good :+1:

**Pull request merged**